test/custom_nextflow.config

```diff
@@ -8,47 +8,50 @@ singularity {
     // cacheDir = ''            // to set a directory where to download the container file 
     autoMounts = true           // to automatically mount host paths in the executed container
 }
-//docker {
-//    enabled = true
-//}
+// docker {
+    // enabled = true
+// }
 
 // CLUSTER SETTINGS:
-executor.name = ''  // e.g. sge
+executor {
+    name = 'local'              // e.g. sge
+    cpus = 2                    // to limit the CPU resources when using local executor
+}
 
 process {
-//  Choose a container type (not needed if all dependencies are locally installed)
-    container = 'docker://vibpsb/ksrates:latest'      // to build a Singularity container
-//  container = 'vibpsb/ksrates:latest'               // to build a Docker container
+    // Choose a container type (not needed if all dependencies are locally installed)
+    container = 'docker://vibpsb/ksrates:latest'        // to build a Singularity container
+    // container = 'vibpsb/ksrates:latest'              // to build a Docker container
 
     // Specify memory and number of threads for processes submitted to the cluster 
     // It is advised to provide higher computational power to wgdParalogs and wgdOrthologs 
     withName: 'wgdParalogs' {
-        cpus =               // e.g. 8
-        penv = ''            // e.g. 'serial'
-        memory = ''          // e.g. '16GB'
-        clusterOptions = ''  // e.g. '-l h_vmem=2G'
+        cpus = 1                // e.g. 8
+        penv = ''               // e.g. 'serial'
+        memory = ''             // e.g. '16GB'
+        clusterOptions = ''     // e.g. '-l h_vmem=2G'
```

Comment:
What happens is these are set to an empty values/string? Should we maybe comment these out since this seems to be pre-configured for a local computer? Same for below.

Reply:
When I ran it, it didn't complain about these fields being set to empty strings (so I guess it ignores them). But I can comment "penv", "memory" and "clusterOptions" out to be more consistent with the "local" pre-setting.

Reply:
Tried that and there is problem: the withName section complains that there are no directives within it! So, either I comment out the whole withName section like below or we leave it as it was. I'll go for the first for now, although the user will have to uncomment quite some stuff if they want to run the test on the cluster (which is however not necessary).
```
    // withName: 'paralogsAnalyses' {
    //     memory = ''             // e.g. '2GB'
    //     clusterOptions = ''     // e.g. '-l h_vmem=2G'
    // }
```

Reply:
Ah, I think that's OK.